keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/api/ArtifactSummary.kt

```diff
@@ -51,7 +51,17 @@ data class ArtifactSummaryInEnvironment(
   val vetoed: ActionMetadata? = null,
   val statefulConstraints: List<StatefulConstraintSummary> = emptyList(),
   val statelessConstraints: List<StatelessConstraintSummary> = emptyList(),
-  val compareLink: String? = null
+  val compareLink: String? = null,
+  val verifications : List<ArtifactVersionEnvironmentVerificationSummary> = emptyList()
+)
+
+data class ArtifactVersionEnvironmentVerificationSummary(
```

Comment:
_Man_, that's a long name, haha! 😄

I've struggled to come up with better names myself in a number of similar cases in keel. I think the problem is that we want to convey all the context of the wrapper objects in these types of classes. In this particular case though, I feel like, since a `Verification`, by definition, only exists for a specific artifact version in a specific environment, maybe we could ditch all that context and just call it `VerificationSummary`?

Reply:
Yeah, that was a crazy long name. Renamed.